### DISCUSSION TITLE
White PDF background as a default

### DISCUSSION THREAD

**Discussion by punkrelique** [hardkoded/puppeteer-sharp#2493] [Q&A]
I have html, that does not have a style for background color. And then, when I try to print it, I get PDF with transparent background. Are there any ways to properly handle it to print with just default white color? Thanks

Looks kinda ugly to set it like this: 
```csharp
await page.EvaluateExpressionAsync("document.body.style.backgroundColor = 'white'");
```

Specs: 
Chromium 122 Stable, Revision: 1241584
Puppeteer v11.0.2

**Comment by kblok** [comment]
Can you share an URL I can test?

**Reply by punkrelique** [reply]
I am using local html files. You can get test html here: https://gist.github.com/punkrelique/6b3bdbb5fc416b6575c08760e8a25274
